modules/gapi/src/streaming/onevpl/cfg_param_device_selector.hpp

```diff
@@ -37,6 +38,7 @@ struct GAPI_EXPORTS CfgParamDeviceSelector final: public IDeviceSelector {
 private:
     Device suggested_device;
     Context suggested_context;
+    std::unique_ptr<Aux> platform_specific_data;
```

Comment:
Abstraction leak, isnt it?

Reply:
?
Would you be kind to clarify your point?